workspace/2023/12/2023-12-numfocus-update.md

```diff
@@ -0,0 +1,24 @@
+## Conda
+
+### Community 
+
+* Subscribe to the new [conda-announce mailing list](https://lists.conda.org/wws/subscribe/announce) to get occasional summaries on the conda ecosystem delivered to your inbox.
+* The Mamba project has [bi-weekly community calls](https://hackmd.io/iOgkxNMpTea6lWczjCov9Q?view) on Tuesdays.
+* The overall conda community has [bi-weekly calls](https://hackmd.io/@conda-community?tags=%5B%22meeting-notes%22%5D) on Wednesdays. 
+
+**Releases**
+
+* **`conda`** [23.11.0 release](https://conda.org/blog/2023-12-07-november-releases/#changes-in-conda-23110) includes 
+  * The speedy `conda-libmamba-solver` as the default dependency solver.  This and other improvements make `conda` run 4-5x faster.
```

Comment:
This was technically part of the 23.10 release, although it looks like we are listing things that have happened recently in the project?

Reply:
```suggestion
  * The speedy `conda-libmamba-solver` as the default dependency solver.  This and other improvements like parallel downloads make `conda` run 4-5x faster.
```

Reply:
@jaimergp We didn't advertise the 23.10 release through NumFOCUS.  My thought is to heap it in here.